dev-tex/latexdiff/latexdiff-1.3.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Compare two latex files and mark up significant differences"
+HOMEPAGE="http://www.ctan.org/tex-archive/support/latexdiff/ https://github.com/ftilmann/latexdiff/"
```

Comment:
https://

Reply:
Also please use `Closes: <url>` rather than `Bug: <url>` in the commit message.

Reply:
done!